tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py35, py36
+envlist = py36, py37, py38
```

Comment:
Please submit as a separte PR, to keep this PR focussed.

Reply:
No reply needed, but some context:

These changes were simply to allow me to more easily test my changes (other than using docker, I only have Python 2.7 and 3.8 on my system). I created PR #16986 which cherry picks these changes.